src/main/java/ch/njol/skript/log/LogEntry.java

```diff
@@ -128,8 +138,32 @@ protected void finalize() {
 	public String toString() {
 		if (node == null || level.intValue() < Level.WARNING.intValue())
 			return message;
+
 		Config c = node.getConfig();
 		return message + from + " (" + c.getFileName() + ", line " + node.getLine() + ": " + node.save().trim() + "')";
 	}
+
+	public String toFormattedString() {
+		if (node == null || level.intValue() < Level.WARNING.intValue())
+			return message;
+
+		Config c = node.getConfig();
+
+		ArgsMessage details;
+		ArgsMessage lineInfo = WARNING_LINE_INFO;
+		if (level.intValue() == Level.WARNING.intValue())  // warnings
+			details = WARNING_DETAILS;
+		else if (level.intValue() == Level.SEVERE.intValue()) { // errors
+			details = ERROR_DETAILS;
+			lineInfo = ERROR_LINE_INFO;
+		}
+		else // anything else
+			details = OTHER_DETAILS;
+
+		return Utils.replaceEnglishChatStyles(
```

Comment:
Could you look into the possibility of escaping chat codes & colour codes if they come from within the scripts? Like if you use `this eff doesnt exist <yellow> stuff` it doesn't format the `<yellow>` as the colour (same with `&e` and `§e`).
Not really needed but would be nice.